partner_preview_scripts/ot_crispr.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+import argparse
+from datetime import datetime
+import gzip
+import json
+import logging
+import sys
+
+import pandas as pd
+
+# The statisticalTestTail is inferred by the column name which is being filtered on:
+FILTER_COLUMN_MAP = {
+    'pos|fdr': 'upper tail',
+    'neg|fdr': 'lower tail',
+    'neg|p-value': 'lower tail',
+    'pos|p-value': 'upper tail'
+}
+
+class OTAR_CRISPR_study_parser(object):
+    def __init__(self, study_url: str) -> None:
+
+        self.study_file = study_url
+
+        # Store the study dataframe after dropping problematic studies:
+        self.study_df = (
+            pd.read_json(study_url)
+
+            # drop rows with no study id or data file:
+            .loc[lambda df: df.studyId.notna() & df.dataFiles.notna()]
+        )
+
+        # Test and warn if multiple studies have the same study id:
+        duplicated_study_ids = self.study_df.loc[lambda df: df.studyId.duplicated()].studyId.tolist()
+        assert len(duplicated_study_ids) == 0, f'Multiple studies have the same study id: {", ".join(duplicated_study_ids)}'
+
+        logging.info(f'Number of studies processed: {len(self.study_df.studyId.unique())}')
+
+        projects = self.study_df.projectId.unique()
+        logging.info(f'Number of projects: {len(projects)} ({", ".join(projects)})')
+
+    def generate_evidence(self, data_folder: str) -> None:
+        """Processing the study dataframe"""
+
+        # Looping through the studies and generating evidence:
+        # Reading all data files and filter for significant hits:
+        study_columns = [
+            'releaseDate',
+            'releaseVersion',
+            'studyId',
+            'dataFiles',
```

Comment:
Once again, maybe I'm reading this wrong, but I don't see a column with that name in the spreadsheet. There, however, `dataFile` (singular)

Reply:
It's now fixed.